Documentation/users/creating-accounts.md

```diff
@@ -0,0 +1,134 @@
+# Creating Tectonic accounts
+
+Use Tectonic Console to create cluster-wide or namespace-specific user, group, or admin accounts.
+
+First, create [Roles][creating-roles] which define rules of access. Then, associate users with Roles using "Role Bindings".
+
+When removed from Tectonic Identity authentication (LDAP or SAML integration), users and groups are cached, and no longer granted the permissions created through Role Bindings.
+
+Use Role Bindings to create the following three user types:
+
+(note - i want to verify these definitions. they're inconsistent, so i'm not sure they are correct)
+
+* **Users:** have access to all common objects within a cluster, but do not have access to change RBAC policies.
+* **Groups:** grant defined access to multiple users. Logged in (authenticated) users are grouped according to their RBAC group memberships.
+
+Permissions granted an individual user are an aggregate of permissions for all roles assigned the user account, and all permissions granted the roles for all groups to which the user belongs.
+
+Individual users may be part of multiple groups.
+
+* **Admins:** have full control over all resources within their cluster or namespace.
+
+##  Prerequisites
+
+User authentication must be enabled before defining user roles. Use Tectonic Identity to integrate with your existing authentication system. For more information, see:
+
+* [Static user management][user-management]
+* [LDAP integration][ldap-integration]
+* [SAML integration][saml-integration]
+
+## Creating Role Bindings
+
+In Tectonic Console, go to *Administration > Role Bindings* and click *Create Binding* to open the *Create Role Binding* page.
+
+1. Select the access level for the Role Binding:
+ * *Namespace Role Bindings* grant access only within the listed namespace.
+ * *Cluster-wide Role Bindings* grant access across the Tectonic cluster.
+While a Cluster Role may be bound down the hierarchy to a Namespace Role Binding, a Namespace Role can't be promoted up the hierarchy to be bound to a Cluster Role Binding.
```

Comment:
I think we can strike this completely. It's more confusing that it seems and it makes sense when you are looking at real data in the UI for roles you have created.